### PR TITLE
test on push display in github

### PR DESCRIPTION
**Pull request by AvshRLev**

### CODE REVIEW

**Comment by AvshRLev** [comment]
[add ci/cd to run those tests automaticly on every push](https://trello.com/c/ulP7KFV7/27-add-ci-cd-to-run-those-tests-automaticly-on-every-push)

File: .gitignore [hunk 1]
```diff
@@ -113,4 +113,8 @@ out
 .yarn/unplugged
 .yarn/build-state.yml
 .yarn/install-state.gz
-.pnp.*
\ No newline at end of file
+.pnp.*
+
+project/pycache/
+static/dist/
+junit.xml
```
Comment (buzzdan) [inline .gitignore:120]: for some reason it doesnt work
i still see the junit.xml in this PR

Comment (buzzdan) [reply .gitignore:120]: maybe try a newline in the end of this line

File: .gitignore [hunk 2]
```diff
@@ -113,4 +113,8 @@ out
 .yarn/unplugged
 .yarn/build-state.yml
 .yarn/install-state.gz
-.pnp.*
\ No newline at end of file
+.pnp.*
+
+project/pycache/
```
Comment (buzzdan) [inline .gitignore:118]: this should be `__pycache__/`

File: .gitignore [hunk 3]
```diff
@@ -113,4 +113,8 @@ out
...
+static/dist/
```
Comment (buzzdan) [inline .gitignore:119]: this should be `dist/`

**Pull request merged**